### ISSUE TITLE
This application is not an Angular application.  Fail on function check I think.

### ISSUE THREAD

**Issue by AlvaPowell** [rangle/augury#1431]
Augury: 1.24.0
Date: Thu, 05 Dec 2019 12:46:42 GMT
OS: Win32

Demo test application:
import { Component, OnInit } from '@angular/core';
declare const getAllAngularTestabilities: Function;
declare const getAllAngularRootElements: Function;
declare const ng: any;

@Component({
  selector: 'app-show-me',
  templateUrl: './show-me.component.html',
  styleUrls: ['./show-me.component.css']
})
export class ShowMeComponent implements OnInit {

  constructor() { 
    const myThing: Function = function add(x,y){return x+y};

    console.log ("getAllAngularTestabilities = " + typeof getAllAngularTestabilities);
    console.log ("getAllAngularRootElements = " + typeof getAllAngularRootElements);  
    console.log ("ng = " + typeof ng);  
    console.log ("myThing = " + typeof myThing);  
  }

  ngOnInit() {
  }

}

Description of issue:
-- Include (clipped) screenshot images if possible.
![image](https://user-images.githubusercontent.com/1615931/70236774-9bccba80-178b-11ea-9e78-35dffa022316.png)
![image](https://user-images.githubusercontent.com/1615931/70236795-a9824000-178b-11ea-9fad-4c1f3b323a46.png)

Angular version (required): ???
Angular CLI: 8.3.20
Node: 13.1.0
OS: win32 x64
Angular: 8.2.14
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.803.20
@angular-devkit/build-angular     0.803.20
@angular-devkit/build-optimizer   0.803.20
@angular-devkit/build-webpack     0.803.20
@angular-devkit/core              8.3.20
@angular-devkit/schematics        8.3.20
@angular/cli                      8.3.20
@ngtools/webpack                  8.3.20
@schematics/angular               8.3.20
@schematics/update                0.803.20
rxjs                              6.4.0
typescript                        3.5.3
webpack                           4.39.2

Steps to reproduce:

1. Create new project using ng new
2. ng serve 
3. connect on browser
4. click augury section in debug

Additional details:
I am getting this error on every Angular project I try Augury on.  Just downloaded and installed all the components in the last week so they are all pretty current.

I created a brand new project using ng new testAugury and then hit enter twice.  Launched this and augury told me it was not an angular project.  Restarted chrome and server same results.

Deleted project ran ng new testAugury again but this time said Y to "add Angular routing?" got same results.

I pulled the source for Augury from Git and found this code in src/backend/utils/app-check.ts
declare const getAllAngularTestabilities: Function;
declare const getAllAngularRootElements: Function;
declare const ng: any;

export const isAngular = () => {
  return typeof getAllAngularTestabilities === 'function'
    && typeof getAllAngularRootElements === 'function';
};

basically put that code into console.log on my project and it logs them as "undefined" (see attached screen shots).  

Tried both chrome and firefox dev edition.
Google Chrome is up to date
Version 78.0.3904.108 (Official Build) (64-bit)
Firefox Browser Developer 72.0b2(64 bit)

Seems like there is something weird going on with the latest stuff or I am just looking at the Augury code completely wrong.  I am completely new to this typescript/javascript/angular world so I very well may be looking at it wrong, or be missing some include or some other special thing I have no knowledge of.

Thanks and hope I wasn't too long winded, being doing dev for a lot of years and more information is most often better.  ;)

**Comment by sumitarora** [comment]
@AlvaPowell This is related to IVY. Can you check if you have IVY enabled in your angular.json?

**Comment by AlvaPowell** [comment]
Sorry hate to be a bone head but I am not even sure how to check that.  

The only thing I can find that references IVY is in node_modules\@schematics\angular\ng-new\schema.json which has this:
    "enableIvy": {
      "description": "When true, creates a new app that uses the Ivy rendering engine.",
      "type": "boolean",
      "default": false

or in node_modules\@angular\compiler-cli\package.json which has this:
 "bin": {
    "ivy-ngcc": "./ngcc/main-ngcc.js",
    "ngc": "./src/main.js",
    "ng-xi18n": "./src/extract_i18n.js"
  },

any help would be appreciated.

**Comment by AlvaPowell** [comment]
This is my Angular.json file:
[angular.zip](https://github.com/rangle/augury/files/3930683/angular.zip)
and all the .json files in the project.
[AllJSON.zip](https://github.com/rangle/augury/files/3930687/AllJSON.zip)

**Comment by yallen011** [comment]
Do you have a link to your code in github?

**Comment by AlvaPowell** [comment]
Just made a repository with it:
https://github.com/AlvaPowell/AuguryTest

**Comment by yallen011** [comment]
Hi Alva, What command do you use to build your code?

**Comment by AlvaPowell** [comment]
I updated to the version 1.24.1 and it seems to be working now.
